### DISCUSSION TITLE
Another Vercel account/project is using a domain I own.

### DISCUSSION THREAD

**Discussion by colebillys19** [vercel/community#6399] [Account Issue] [answered]
### Question

I purchased a domain for one of my Vercel projects (via Namecheap). When I went to connect the domain to my project I got the following:

"Another Vercel account is using this domain. Set the following TXT record on _vercel.mydomain.app to use www.mydomain.app in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed."

I'm confused as to how another account is using the domain if I own it. Is it that another person previously owned it and it's still associated with one of their projects? I'm happy to prove ownership of the domain. Please help!

Thanks.

**Comment by colebillys19** [comment, marked as answer]
I was able to figure this out. I just had to configure A and CNAME records on Namecheap. I suggest accounting for this situation and updating the messaging so it's not "Another Vercel account is using this domain..."

**Comment by Bajingan1q** [comment]
> ### Question
> I purchased a domain for one of my Vercel projects (via Namecheap). When I went to connect the domain to my project I got the following:
> 
> "Another Vercel account is using this domain. Set the following TXT record on _vercel.mydomain.app to use [www.mydomain.app](http://www.mydomain.app) in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed."
> 
> I'm confused as to how another account is using the domain if I own it. Is it that another person previously owned it and it's still associated with one of their projects? I'm happy to prove ownership of the domain. Please help!
> 
> Thanks.
